tests/unit/amo/components/TestSearchContextCard.js

```diff
@@ -104,4 +109,125 @@ describe('SearchContextCard', () => {
     expect(root.find('.SearchContextCard-header'))
       .toIncludeText('No add-ons found');
   });
+
+  // addonType = ADDON_TYPE_THEME
+  it('should render singular form when only one result is found with addonType as ADDON_TYPE_THEME', () => {
+    const { store } = dispatchSearchResults({
+      addons: { [fakeAddon.slug]: fakeAddon },
+      filters: {
+        addonType: ADDON_TYPE_THEME,
+        query: 'test',
+      },
+    });
+
+    const root = render({ store });
+
+    expect(root.find('.SearchContextCard-header'))
+      .toIncludeText('1 theme found for "test"');
+  });
+
+  it('should render plural form when multiple results are found with addonType as ADDON_TYPE_THEME', () => {
+    const { store } = dispatchSearchResults({
+      filters: {
+        addonType: ADDON_TYPE_THEME,
+        query: 'test',
+      },
+    });
+
+    const root = render({ store });
+
+    expect(root.find('.SearchContextCard-header'))
+      .toIncludeText('2 themes found for "test"');
+  });
+
+  // addonType = ADDON_TYPE_DICT
+  it('should render singular form when only one result is found with addonType as ADDON_TYPE_DICT', () => {
```

Comment:
I think you can just end with `ADDON_TYPE_DICT` instead of `addonType as ADDON_TYPE_DICT`. Similar for the rest of the test specs.

They're otherwise really good though, thanks for writing really clear test specs! 👍